authors/apps/notifications/serializers.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import serializers
+
+from .models import Notification
+
+from authors.apps.articles.serializers import ArticleSerializer
+from django.utils.timesince import timesince
+
+
+class NotificationSerializer(serializers.ModelSerializer):
+    article = ArticleSerializer('article')
+    timestance = serializers.SerializerMethodField(method_name='timesince')
+    unread = serializers.SerializerMethodField(method_name='read')
+
+    class Meta:
+        """
+        Notification fields to be returned to users
+        """
+        model = Notification
+        fields = ('unread', 'created_at', 'notification', 'classification',
+                  'article', 'timestance')
+
+    def timesince(self, instance, now=None):
```

Comment:
This method name can be improved further. For example, it could be named `get_timesince`. This way, it informs other users that it is going to return the time difference as it does. It can also be named `calculate_timesince`. I suggest you make it more descriptive.